starter-code/functions-and-arrays.js

```diff
@@ -63,7 +97,12 @@ console.log(averageLength);
 
 // Unique Arrays
 function uniquifyArray (array) {
-
+  var arrayVacio = [];
+  for (var i = 0; i < array.length; i++){
+    if (indexOf(array[i]) == -1) {
+      push...
```

Comment:
Sobre la campana!!! Buen trabajo chicos, os habeís tomado vuestro tiempo y habéis afianzado conocimientos. Seguid trabajando así!